MekHQ/src/mekhq/campaign/stratcon/StratconCampaignState.java

```diff
@@ -176,6 +176,20 @@ public void useSupportPoint() {
         supportPoints--;
     }
 
+    /**
+     * Decreases the number of support points by the specified increment.
+     *
+     * @param increment The number of support points to use/decrease.
+     */
+    public void useSupportPoints(int increment) {
+        supportPoints -= increment;
+    }
+
+    public void convertVictoryToSupportPoint() {
+        victoryPoints--;
+        supportPoints++;
+    }
```

Comment:
lacking check if VP are greater than zero.

Reply:
I'm glad you spotted this, because that whole method should have been removed when merging in master.